data/remnant missions.txt

```diff
@@ -493,5 +592,771 @@ mission "Remnant: Technology Available"
+	minor
+	source Viminal
+	to offer
+		has "license: Remnant"
+		random < 50
+	on offer
+		require "Catalytic Ramscoop"
+		conversation
+			`As you are walking through the spaceport, a woman comes up to you and introduces herself as an engineer at one of the Remnant development labs. She pauses awkwardly before beginning to chant "As your ship is the only one we have seen in generations that has outfits from ancestral space, we are intrigued by it. Could we take a closer look?"`
+			choice
+				`	"Yes, you are welcome to examine it."`
+					goto deep
+				`	"No, stay away from my ship."`
+					decline
+				`	"Yes, but don't touch anything."`
+
+			`	"I'll be careful," she sings with a tinge of sadness. "I am an engineer, not a Korath."`
+				goto more
+
+			label deep
+			`	"Fascinating," she sings as she clambers over <ship> with the grace of someone who obviously knows her ships. "Above all else, I am an engineer of movement," she chants. "We have done well to hide as deep in the Ember as we have, but our ship's limited range can take us no farther. Examining systems from elsewhere may inspire new developments."`
+
+			label more
+			`	After examining <ship> for quite some time, she stops beneath it and looks up. "Is this a new form of ramscoop?" She asks as she gestures at your ship's Ramscoop. As you nod in the affirmative, she continues. "As I said before, fuel has always been a great limiter of our exploration," says the woman. "If you could get a couple extra that we could dismantle, this could be very useful to us."`
+			choice
+				`	"Sure, I could bring you some. What are you offering in return?"`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a bit of haggling, she agrees to pay you one million credits in exchange for a cargo of two Deep "Catalytic Ramscoops." Given how cheap they are to purchase in Deep space, you will be earning a very tidy profit on the deal.`
+				accept
+	
+	on complete
+		outfit "Catalytic Ramscoop" -2
+		payment 1000000
+		conversation
+			`You visit the local shipyard and eventually find the engineer busily repairing a damaged Starling. You inform her that you have brought a pair of "Catalytic Ramscoops" for her to study. She quickly tightens a couple more connectors and jumps down from her perch, gesturing at an assistant to transfer you the credits you were promised. "Thank you! Our maps of human space are old, and if they are still accurate, you traveled a very long way to bring these to us." Her chanting voice echoes off the starling above you with an interesting resonance.`
+			`	"Thank you for your help," she says. "If we can unlock the secrets of these scoops, it will give our ships greater range as they search for a safer refuge." As she picks up the remote control for the small loading truck with the two ramscoops she trills at you over her shoulder. "Oh, my name is Tali. I'll let you know what I find."`
```

Comment:
I originally wanted all three of them to be doable, but as you pointed out, they don't have the branching to alter the conversation if one has already met Tali. So for now they have been switched to being mutually exclusive.

I'd like to do a PR to introduce branching dialogue so that all three are doable later on, but I that's a project for later. Having them be mutually exclusive is simpler and a more straight forward path for now.

Reply:
Fixed the double thanks and referring to them as scoops.